jvm-libs/linea/testing/l1-blob-and-proof-submission/src/main/kotlin/net/consensys/linea/testing/submission/SubmissionTestHelper.kt

```diff
@@ -1,50 +1,102 @@
 package net.consensys.linea.testing.submission
 
+import build.linea.domain.BlockInterval
 import linea.web3j.waitForTxReceipt
 import net.consensys.zkevm.coordinator.clients.smartcontract.LineaRollupSmartContractClient
 import net.consensys.zkevm.domain.Aggregation
+import net.consensys.zkevm.domain.BlobRecord
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
 import org.web3j.protocol.Web3j
+import tech.pegasys.teku.infrastructure.async.SafeFuture
 import kotlin.time.Duration
 import kotlin.time.Duration.Companion.minutes
 
+fun assertTxSuccess(
+  txHash: String,
+  interval: BlockInterval,
+  submissionType: String,
+  l1Web3jClient: Web3j,
+  timeout: Duration = 1.minutes
+) {
+  l1Web3jClient.waitForTxReceipt(
+    txHash = txHash,
+    timeout = timeout
+  ).also { txReceipt ->
+    if (txReceipt.status != "0x1") {
```

Comment:
Why aren't we using assertj's `Assertions` here? I believe they can help making this code a little more concise